R/summary_funs_formula.R

```diff
@@ -0,0 +1,33 @@
+.get_sub_summaries_poc <- function(submodels, test_points, refmodel, family_kl, groups=NULL) {
```

Comment:
Right now there would a conflict of names in the package, so I kept them to avoid removing the other files directly, but I agree that we should decide something better, either drop the current code (so we don't have duplicated code) or to choose a more elegant suffix.